### PR TITLE
feat: add prefix for email subject and heading for privatecloud

### PR DESCRIPTION
**Pull request by calbertan**
image

### CODE REVIEW

**Comment by calbertan** [comment]
image

**Review by junminahn — COMMENTED** [review]
Two considerations:

`Email Subject`:
- To avoid duplicates, we can think about adding the prefix in `sendEmail` function.

`Content Header in the Email Body`:
- It might be easy to forget to include that specific word when updating or adding new email templates. So, unless there's a way to adding the content header's prefix within `sendEmail` function (similar to what we do for the subject), it's better not to add it in the header.

So, ideally, we want to touch a single file that includes `sendEmail` function to meet the requirement we have!

File: emails/templates/private-cloud/AdminCreateRequest.tsx [hunk 1]
```diff
@@ -7,15 +7,13 @@ import { Tailwind } from '@react-email/tailwind';
...
 import { TailwindConfig } from '../../components/TailwindConfig';
-const defaultTheme = require('tailwindcss/defaultTheme');
 
 interface EmailProp {
   request: PrivateCloudRequestWithRequestedProject;
 }
 
 export const NewRequestTemplate = ({ request }: EmailProp) => {
   if (!request) return <></>;
-
```
Comment (junminahn) [inline emails/templates/private-cloud/AdminCreateRequest.tsx:18]: to reduce the unnecessary code changes and also for readability, let's keep this new line.

File: emails/templates/private-cloud/CreateRequest.tsx [hunk 2]
```diff
@@ -7,15 +7,13 @@ import { Tailwind } from '@react-email/tailwind';
...
 import { TailwindConfig } from '../../components/TailwindConfig';
-const defaultTheme = require('tailwindcss/defaultTheme');
 
 interface EmailProp {
   request: PrivateCloudRequestWithRequestedProject;
 }
 
 export const NewRequestTemplate = ({ request }: EmailProp) => {
   if (!request) return <></>;
-
```
Comment (junminahn) [inline emails/templates/private-cloud/CreateRequest.tsx:18]: to reduce the unnecessary code changes and also for readability, let's keep this new line.

File: emails/templates/private-cloud/RequestApproval.tsx [hunk 3]
```diff
@@ -14,7 +14,6 @@ interface EmailProp {
 
 export const RequestApprovalTemplate = ({ request }: EmailProp) => {
   if (!request) return <></>;
-
```
Comment (junminahn) [inline emails/templates/private-cloud/RequestApproval.tsx:17]: to reduce the unnecessary code changes and also for readability, let's keep this new line.

File: emails/templates/private-cloud/RequestRejection.tsx [hunk 4]
```diff
@@ -11,7 +11,6 @@ interface EmailProp {
 }
 export const RequestRejectionTemplate = ({ productName, comment }: EmailProp) => {
   if (!productName) return <></>;
-
```
Comment (junminahn) [inline emails/templates/private-cloud/RequestRejection.tsx:14]: to reduce the unnecessary code changes and also for readability, let's keep this new line.

**Comment by junminahn** [comment]
@calbertan I left very minor comments!

**Review by junminahn — APPROVED** [review]

**Pull request merged**